app/config.py

```diff
@@ -438,87 +438,54 @@ class Config(object):
         # app/celery/nightly_tasks.py
         "timeout-sending-notifications": {
             "task": "timeout-sending-notifications",
-            "schedule": crontab(hour=0, minute=5),
+            "schedule": crontab(hour=5, minute=5),  # 00:05 EST in UTC
```

Comment:
When the clock rolls back 1 hour on May 10th could this end up scheduling the task for 11:05PM EST the day before?
Otherwise looks good!

Reply:
yes. I believe that is exactly what will happen

Reply:
Are we ok with that or should we use something like `pytz` to check the current DST status and adjust for consistent scheduling?

Reply:
I am okay with us not having a consistent schedule on one day, I cant think of downsides here

Reply:
Sounds good to me then!